packages/puppeteer-core/src/node/ScreenRecorder.ts

```diff
@@ -183,21 +215,39 @@ export class ScreenRecorder extends PassThrough {
           // Sets the format
           ['-f', 'webm'],
           // Sets the quality. Lower the better.
-          ['-crf', `${CRF_VALUE}`],
+          ['-crf', `${quality}`],
           // Sets the quality and how efficient the compression will be.
           ['-deadline', 'realtime', '-cpu-used', '8'],
         ].flat();
       case 'gif':
+        fps = DEFAULT_FPS === fps ? 20 : 'fps=source_fps';
+        if (!Number.isInteger(loop) || loop === 0) loop = ~~!loop * -1;
```

Comment:
Can't we just set it to the correct default where it won't loop? Rather then this evaluation?